pkg/services/guardian/guardian.go

```diff
@@ -213,6 +214,38 @@ func (g *dashboardGuardianImpl) getTeams() ([]*models.TeamDTO, error) {
 	return query.Result, err
 }
 
+func (g *dashboardGuardianImpl) AddHiddenPermissions(updatePermissions []*models.DashboardAcl, cfg *setting.Cfg) ([]*models.DashboardAcl, error) {
```

Comment:
I'm wondering if we want to move this out of dashboard guardian. It's very specific to what we are doing right now and doesn't fit with what the guardian is doing well. What do you think about replacing it with a `GetHiddenAcls`-function? It would do almost the same thing but not add the permissions. It means the guardian gets to stay focused on getting permissions and checking permissions instead of adding a third job.